gossipd/routing.c

```diff
@@ -892,21 +892,19 @@ void handle_channel_update(struct routing_state *rstate, const u8 *update)
 
 	c = chan->connections[direction];
 
-	/* When we local_add_channel(), we only half-populate, so this case
-	 * is possible. */
+	/* Channel could have been pruned: re-add */
```

Comment:
This is for the case that we had a channel, it got pruned, but we're still kept the `routing_channel` due to the other direction being updated? In that case this is much better, I agree.


The drop is a leftover from the times we considered half-channels independently from each other.